styles/03-dusk.json

```diff
@@ -60,6 +60,7 @@
 			"letterSpacing": "-0.18px",
 			"lineHeight": "1.5"
 		},
+		"css": ".wp-block-heading.is-style-text-annotation {padding-bottom:0.15rem;}",
```

Comment:
I still see a bit more spacing at the bottom. In my testing I believe approaching `0.1rem` makes it look better. I'll defer to @beafialho to decide the value here.

Reply:
Ok, I'll make any additional changes necessary once we hear back

Reply:
@juanfra stepped away for a but, just following up on a few items. Are we still waiting for confirmation on this PR?

Reply:
@troychaplin thanks for coming back to this one. Unfortunately, as mentioned in the review, the PR will have the side effect of keeping/persisting the CSS once saved, affecting styles when changing variation. We need to find a solution that doesn't have this type of side effects. If we find it, we could confirm the padding-bottom of the annotation style for that variation in particular.

Reply:
Ok it sounds like it would be best to just close this PR and remove my name from the issue. I will still take some time to look at other solutions, but best to leave it unassigned in case someone else wants to take a look as well.

Reply:
Ok, thank you Troy!